src/pytorch_lightning/plugins/io/checkpoint_plugin.py

```diff
@@ -60,3 +60,14 @@ def remove_checkpoint(self, path: _PATH) -> None:
         Args:
             path: Path to checkpoint
         """
+
+    @property
+    def is_wrapper(self) -> bool:
+        """Property that tells whether a ``CheckpointIO`` is a wrapper around another ``CheckpointIO`` object."""
+        from pytorch_lightning.plugins.io.wrapper import _WrappingCheckpointIO
+
+        return isinstance(self, _WrappingCheckpointIO)
```

Comment:
IMO this property shouldn't exist here, and the places that call this should use `isinstance` directly